### PR TITLE
api doc's param & code-block display correctly

### PR DESCRIPTION
**Pull request by 2bndy5**
I went to read the docs for this library and noticed all the parameters and example code-blocks were not displaying properly because there was no empty line that preceded them. Verified locally via `sphinx-build` that the changes generate properly.

Upon further reading I noticed there was a (un-admonished) deprecation notice and a reference to an example that was not being displayed on RTD. I added the example to `examples.rst` as "Basic Forever Loop", and I changed the notice to an admonished "Note" so that it sticks out to the reader better.

Side-note: there are a lot more examples in this library that are not being displayed on RTD. Furthermore there are no module-level docstrings that would help identify the purpose of the examples (other than the example filename). I'm not too familiar with this library, so I didn't go crazy adding all the examples to `examples.rst`.

### CODE REVIEW

File: adafruit_minimqtt/adafruit_minimqtt.py [hunk 1]
```diff
@@ -114,7 +115,6 @@ class MQTT:
     :param bool is_ssl: Sets a secure or insecure connection with the broker.
     :param bool log: Attaches a logger to the MQTT client, defaults to logging level INFO.
     :param int keep_alive: KeepAlive interval between the broker and the MiniMQTT client.
-
```
Comment (brentru) [inline adafruit_minimqtt/adafruit_minimqtt.py:117]: This blank line should remain here
> Insert a blank line after all docstrings (one-line or multi-line) that document a class
 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

File: adafruit_minimqtt/adafruit_minimqtt.py [hunk 2]
```diff
@@ -193,19 +193,23 @@ def __exit__(self, exception_type, exception_value, traceback):
         self.deinit()
 
     def deinit(self):
-        """De-initializes the MQTT client and disconnects from
-        the mqtt broker.
-
+        """De-initializes the MQTT client and disconnects from the mqtt broker.
```
Comment (brentru) [inline adafruit_minimqtt/adafruit_minimqtt.py:196]: Can this fit on one line instead?

File: adafruit_minimqtt/adafruit_minimqtt.py [hunk 3]
```diff
@@ -300,8 +303,7 @@ def connect(self, clean_session=True):
                 raise MMQTTException("Invalid broker address defined.", e)
 
         # Fixed Header
-        fixed_header = bytearray()
-        fixed_header.append(0x10)
+        fixed_header = bytearray([0x10])
```
Comment (brentru) [inline adafruit_minimqtt/adafruit_minimqtt.py:306]: Could we do this in a separate PR instead? I like it, but it shouldn't be included with this PR.

File: adafruit_minimqtt/adafruit_minimqtt.py [hunk 4]
```diff
@@ -668,11 +692,11 @@ def loop_forever(self):
         method if you want to run a program forever.
         Code below a call to this method will NOT execute.
 
-        NOTE: This method is depreciated and will be removed in the
-        next major release. Please see examples/minimqtt_pub_sub_blocking.py
-        for an example of creating a blocking loop which can handle wireless
-        network events.
-
+        .. note:: This method is depreciated and will be removed in the
```
Comment (brentru) [inline adafruit_minimqtt/adafruit_minimqtt.py:695]: > I noticed there was a (un-admonished) deprecation notice and a reference to an example that was not being displayed on RTD.

In the next PR (with the `fixed_header` change), could you also remove this method?

Comment (2bndy5) [reply adafruit_minimqtt/adafruit_minimqtt.py:695]: Yes; can do. Prepping for the next "major" release? No planned "minor" releases in immediate future? Is the [Bluetooth PR](#15 ) such a significant change? I also see you have a branch for compatibility with CPython sockets...

File: adafruit_minimqtt/adafruit_minimqtt.py (hunk 3 above)
Comment (2bndy5) [reply adafruit_minimqtt/adafruit_minimqtt.py]: ah, you caught me... good eye. Opened a new PR with this change

File: adafruit_minimqtt/adafruit_minimqtt.py (hunk 1 above)
Comment (2bndy5) [reply adafruit_minimqtt/adafruit_minimqtt.py]: Personally I'm not a fan of the black linter but request granted.

File: adafruit_minimqtt/adafruit_minimqtt.py (hunk 4 above)
Comment (2bndy5) [reply adafruit_minimqtt/adafruit_minimqtt.py:696]: Opened #45 to satisfy request

Comment (brentru) [reply adafruit_minimqtt/adafruit_minimqtt.py:696]: @2bnd5 re: I'm not ready to merge the CPython sockets branch just yet, but yes, that'll be a major release.
